### PR TITLE
disable addon

### PR DESCRIPTION
**Pull request by dead-horse**
must disable addon after _findPath

### CODE REVIEW

**Comment by fengmk2** [comment]
What's the different?

**Comment by dead-horse** [comment]
这个测试用例不修改代码就跑不过

File: test/fixtures/require_addon.js [hunk 1]
```diff
@@ -1,2 +1,2 @@
-var microtime = require('./addons.node');
+var microtime = require('./addons');
```
Comment (fengmk2) [inline test/fixtures/require_addon.js:1]: 明白了, 就是说没必要写.node 后缀的.

**Comment by fengmk2** [comment]
+1

**Pull request merged**